exercises/atbash-cipher/example.ml

```diff
@@ -0,0 +1,57 @@
+let substitute = function
+  | 'a' .. 'z' as c ->
+    let offset = (Char.code c) - (Char.code 'a') in
+    let code = (Char.code 'z') - offset in
+    Char.chr code
+  | other -> other
+
+let is_encodable = function
+  | 'a' .. 'z' | '0' .. '9' -> true
+  | _ -> false
+
+let explode s =
```

Comment:
I don't know of a way of providing a signature in this case. Either rewrite if you think adding a signature is worth it, or leave as is.